src/module-1/splitAndMerge/index.js

```diff
@@ -1 +1,4 @@
-export const splitAndMerge = (str = "", separator = "") => {};
+export const splitAndMerge = (str = "", separator = "") => {
+  return str.split(" ").map(str => str.split("").join(separator)).join(" ");
```

Comment:
```suggestion
  return str.split(" ").map(word => word.split("").join(separator)).join(" ");
```  
avoid using shadowed variable names, it makes readability and debugging a hard  work